addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -720,6 +720,21 @@ blockquote {
                 @include button-outline-variant($-btn-secondary-border-color);
             }
 
+            // Custom
+            .btn-custom:hover, .btn-fill-custom:hover {
+                filter: brightness(0.8);
```

Comment:
Maybe worth trying invert instead/in combination. Otherwise this has no effect on black buttons

Reply:
I tried a few combinations, up to three filters, but there is always a color which gets translated to itself.
Given that black buttons are likely, I replaced the `brightness(0.8)` by an `invert(0.2)`.
This makes gray 50% a fixed color, but even with combinations some shade will end up remaining unchanged.
We would need the user to select "Dark custom" or "Light custom" to be able to apply distinct filters...